Autopilot/SensorFusion/Inc/SensorFusion.hpp

```diff
@@ -65,8 +64,8 @@ typedef struct
     long double longitude; // 8 Bytes
     float utcTime;     // 4 Bytes. Time in seconds since 00:00 (midnight)
     float groundSpeed; // in m/s
-    short heading; // in degrees. Should be between 0-360 at all times, but using integer just in case
     int altitude; // in m
+    short heading; // in degrees. Should be between 0-360 at all times, but using integer just in case
```

Comment:
comment does not conform to datatype used. 

Also, the GPS struct uses int for heading. For consistency, we should use `int` here too